packages/nimbus/src/type-utils/shared-types.ts

```diff
@@ -21,12 +21,6 @@ export type { DateValue } from "@internationalized/date";
  */
 export type { TimeValue } from "react-aria";
 
-/**
- * Key type for collection items from react-aria-components
- * Used by: Select, ComboBox, Menu, TagGroup, DataTable, and other collection components
- */
-export type { Key } from "react-aria-components";
-
```

Comment:
@jaikamat & @stephsprinkle  also made this adjustment in be61f2ca5ec2ab107a2c603ac531ef7e831a447b as discussed